.github/workflows/nightly.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     uses: ./.github/workflows/_linux-build.yml
     needs: get-label-type
     with:
-      runner: "${{ needs.get-label-type.outputs.label-type }}linux.2xlarge"
+      runner: "${{ needs.get-label-type.outputs.label-type }}linux.8xlarge.memory"
```

Comment:
It's is poorly named, this job is actually the job that builds the pytorch binary, not the one that builds the docs.  The job that builds the docs should be in _docs.yml